src/main/scala/io/iohk/ethereum/consensus/ethash/EthashBlockCreator.scala

```diff
@@ -45,10 +44,13 @@ class EthashBlockCreator(
     }
   }
 
-  private def getOmmersFromPool(parentBlockHash: ByteString): Future[OmmersPool.Ommers] = {
-    (ommersPool ? OmmersPool.GetOmmers(parentBlockHash))(Timeout(miningConfig.ommerPoolQueryTimeout))
-      .mapTo[OmmersPool.Ommers]
-      .recover { case ex =>
+  private def getOmmersFromPool(parentBlockHash: ByteString): Task[OmmersPool.Ommers] = {
```

Comment:
IMHO there is duplication with getOmmersFromPool in EthService
possibly to be fix'ed on separate PR

Can we use simlar approach: ommersPool
        .askFor[OmmersPool.Ommers](...)